src/styles/base.css

```diff
@@ -42,3 +41,8 @@
 [role=banner] [data-icon='icon-claudy'] {
   background-image: url(../assets/icons/16/icon-claudy.svg)
```

Comment:
Missing `;`

Reply:
? `;` are missing everywhere in this file. It does not matter, it is on the last line.

Reply:
Anyway, I added them in https://github.com/cozy/cozy-bar/pull/133/commits/a22bfd1d8d721a9d4470db35afc3ffcc6a4ae8b9